stdlib/public/runtime/EnvironmentVariables.cpp

```diff
@@ -115,7 +115,7 @@ static uint32_t parse_uint32_t(const char *name,
   }
   if (n > UINT32_MAX) {
     swift::warning(RuntimeErrorFlagNone,
-                   "Warning: %s=%s out of bounds, clamping to %d.\n",
+                   "Warning: %s=%s out of bounds, clamping to %u.\n",
```

Comment:
Can we use `PRIu32` from `inttypes.h` here?

Reply:
done